flink-runtime-web/web-dashboard/web/js/vendor.js

```diff
@@ -15,11 +15,11 @@ ua.range=function(t,e,n){if(arguments.length<3&&(n=1,arguments.length<2&&(e=t,t=
 }}function Ln(t){return function(e,n,r,i){var o=Wn(r);return null==r&&o===xe?rn(e,n,t):on(e,n,o(r,i,1),t)}}function Pn(t,e,n,r,i,o,a,s){var u=e&M;if(!u&&"function"!=typeof t)throw new Zo(z);var l=r?r.length:0;if(l||(e&=~(A|T),r=i=$),l-=i?i.length:0,e&T){var c=r,f=i;r=i=$}var d=u?$:Wa(t),h=[t,e,n,r,i,c,f,o,a,s];if(d&&(ir(h,d),e=h[1],s=h[9]),h[9]=null==s?u?0:t.length:$a(s-l,0)||0,e==_)var p=pn(h[0],h[2]);else p=e!=A&&e!=(_|A)||h[4].length?Dn.apply($,h):jn.apply($,h);var g=d?za:Ba;return g(p,h)}function Fn(t,e,n,r,i,o,a){var s=-1,u=t.length,l=e.length;if(u!=l&&!(i&&l>u))return!1;for(;++s<u;){var c=t[s],f=e[s],d=r?r(i?f:c,i?c:f,s):$;if(d!==$){if(d)continue;return!1}if(i){if(!de(e,function(t){return c===t||n(c,t,r,i,o,a)}))return!1}else if(c!==f&&!n(c,f,r,i,o,a))return!1}return!0}function qn(t,e,n){switch(n){case V:case H:return+t==+e;case U:return t.name==e.name&&t.message==e.message;case X:return t!=+t?e!=+e:t==+e;case Q:case J:return t==e+""}return!1}function zn(t,e,n,r,i,o,a){var s=zs(t),u=s.length,l=zs(e),c=l.length;if(u!=c&&!i)return!1;for(var f=u;f--;){var d=s[f];if(!(i?d in e:ea.call(e,d)))return!1}for(var h=i;++f<u;){d=s[f];var p=t[d],g=e[d],v=r?r(i?g:p,i?p:g,d):$;if(!(v===$?n(p,g,r,i,o,a):v))return!1;h||(h="constructor"==d)}if(!h){var m=t.constructor,y=e.constructor;if(m!=y&&"constructor"in t&&"constructor"in e&&!("function"==typeof m&&m instanceof m&&"function"==typeof y&
```

Comment:
these changes should be reverted (these are usually caused by using different versions to build the web UI and generally just clutter up the diff.)